site/app/templates/plagiarism/PlagiarismConfigurationForm.twig

```diff
@@ -22,167 +23,217 @@
+                                <input type="checkbox" id="regex_submissions_dir" name="regex_dir[]" value="submissions" {{ "submissions" in regex_dirs ? "checked" : ""}} />
+                                <label for="regex_submissions_dir">Submissions</label>
+                            </span>
+                            <span class="radio-label-pair">
+                                <input type="checkbox" id="regex_results_dir" name="regex_dir[]" value="results" {{ "results" in regex_dirs ? "checked" : "" }} />
+                                <label for="regex_results_dir">Results</label>
+                            </span>
+                            <span class="radio-label-pair">
+                                <input type="checkbox" id="regex_checkout_dir" name="regex_dir[]" value="checkout" {{ "checkout" in regex_dirs ? "checked" : "" }} />
+                                <label for="regex_checkout_dir">Checkout</label>
+                            </span>
+                        </span>
+                        <input type="text" id="regex_to_select_files" name="regex_to_select_files" class="low-margin-top" value="{{ regex }}" placeholder="Leave blank to select all files"/>
+                    </div>
                 </div>
             </div>
+            {##################################################################}
             <div class="plag-data-group">
                 <div class="plag-data-label">Language:</div>
                 <div class="plag-data">
                     <select name="language">
+                        <option value="plaintext" {{ language["plaintext"] }}>Plain Text</option>
                         <option value="python" {{ language["python"] }}>Python</option>
                         <option value="cpp" {{ language["cpp"] }}>C++</option>
                         <option value="java" {{ language["java"] }}>Java</option>
-                        <option value="plaintext" {{ language["plaintext"] }}>Plain Text</option>
                         <option value="mips" {{ language["mips"] }}>MIPS Assembly</option>
                     </select>
                 </div>
             </div>
+            {##################################################################}
             <div class="plag-data-group">
-                <div class="plag-data-label">Threshold/Maximum number of students (more than this number of students with matching code will be considered common code):</div>
+                <div class="plag-data-label">Common Code Threshold:</div>
+                <label for="threshold">More than this number of students with matching code will be considered common code</label>
                 <div class="plag-data">
-                    <input type="text" name="threshold" value="{{ threshold }}" placeholder="(Required)"/>
+                    <input type="text" id="threshold" name="threshold" value="{{ threshold }}" placeholder="(Required)"/>
```

Comment:
This should be a `type="number"` input. Can also set a `min` value of `2` here to help guide users on what the minimum value is. Should also put it in the description as well.